g2p_programs/models/cycle.py

```diff
@@ -72,6 +78,18 @@ def _compute_entitlements_count(self):
                 entitlements_count = len(rec.entitlement_ids)
             rec.update({"entitlements_count": entitlements_count})
 
+    @api.depends("entitlement_ids")
+    def _compute_payments_count(self):
+        for rec in self:
+            payments_count = 0
+            if rec.entitlement_ids:
+                payments = self.env["g2p.payment"].search(
+                    [("entitlement_id", "in", rec.entitlement_ids.ids)]
+                )
+                if payments:
+                    payments_count = len(payments)
```

Comment:
Can we use `search_count` instead of `search`?

Reply:
Done